### ISSUE TITLE
[SDKMAN!] SDKMAN! suggests alpha & beta versions of Quarkus as default

### ISSUE THREAD

**Issue by ksilz** [quarkusio/quarkus#32260]
### Describe the bug

SKDMAN! suggests alpha and beta versions of Quarkus as the default Quarkus versions. It only should offer stable versions.

I filed this as [an SDKMAN! issue](https://github.com/sdkman/sdkman-cli/issues/1175) but was told to report it with Slack. There @maxandersen [told me to file a Quarkus issue instead](https://sdkman.slack.com/archives/C02PF215HJ9/p1680087495300379).

### Expected behavior

SKDMAN! offers only stable versions of Quarkus as the default.

### Actual behavior

SKDMAN! offers alpha and beta versions of Quarkus as the default, if they're the last ones published.

### How to Reproduce?

1. Install SKDMAN! on a Mac.
2. Install Quarkus through SDKMAN!.
3. Wait for a Quarkus 3.0 Alpha/beta version to appear.
4. Run these SDKMAN! commands:
```
sdk selfupdate
sdk update
sdk ug
```

[In the past](https://github.com/sdkman/sdkman-cli/issues/1175), SKDMAN! offered an alpha version.
```
Available defaults:
-e quarkus (local: 2.16.1.Final, 2.16.2.Final; default: 3.0.0.Alpha4)

-e -n Use prescribed default version(s)? (Y/n):
```

Today, it offers the beta version:
![image (4)](https://user-images.githubusercontent.com/1061209/228794866-493dc79d-e85f-46b9-82c7-33457fb47301.png)

### Output of `uname -a` or `ver`

Darwin m1x-1679.local 22.4.0 Darwin Kernel Version 22.4.0: Mon Mar  6 20:59:28 PST 2023; root:xnu-8796.101.5~3/RELEASE_ARM64_T6000 arm64

### Output of `java -version`

openjdk version "17.0.6" 2023-01-17 OpenJDK Runtime Environment Temurin-17.0.6+10 (build 17.0.6+10) OpenJDK 64-Bit Server VM Temurin-17.0.6+10 (build 17.0.6+10, mixed mode)

### GraalVM version (if different from Java)

openjdk version "17.0.5" 2022-10-18 OpenJDK Runtime Environment GraalVM CE 22.3.0 (build 17.0.5+8-jvmci-22.3-b08) OpenJDK 64-Bit Server VM GraalVM CE 22.3.0 (build 17.0.5+8-jvmci-22.3-b08, mixed mode, sharing)

### Quarkus version or git rev

2.16.5.Final

### Build tool (ie. output of `mvnw --version` or `gradlew --version`)

Gradle 7.6.1
Build time:   2023-02-24 13:54:42 UTC
Revision:     3905fe8ac072bbd925c70ddbddddf4463341f4b4 
Kotlin:       1.7.10
Groovy:       3.0.13
Ant:          Apache Ant(TM) version 1.10.11 compiled on July 10 2021
JVM:          17.0.5 (GraalVM Community 17.0.5+8-jvmci-22.3-b08)
OS:           Mac OS X 13.3 aarch64

**Comment by ksilz** [comment]
I can't tell you. I have to wait for a new Beta to arrive.

**Comment by maxandersen** [comment]
Hmm. 

Would this not be sufficient ? 

'sdk uninstall quarkus; sdk install quarkus'

**Comment by ksilz** [comment]
You're mostly right! Here's how I tested it:
```
sdk rm quarkus 2.16.5.Final
skd i quarkus
```

You have to specify a version to remove, even if you just have one installed.

Anyhow, your fix worked: I got Quarkus 2.16.5.Final as the default version. Thank you!
